src/main/webapp/destinationCreation.html

```diff
@@ -3,11 +3,18 @@
   <head>
     <meta charset="UTF-8"/>
     <title>Destination Creation</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
     <link rel="stylesheet" href="destinationCreation.css"/>
+    <script src="destinationCreation.js"></script>
+    <script src="config.js"></script>
```

Comment:
I think I read somewhere that JS script tags should be placed in the order that they're used (i.e. since destinationCreation.js uses config.js, config.js should go first). However, if this runs correctly, then this probably doesn't matter too much.

Reply:
Interesting. I had never heard of that. I will switch them to fit this ordering.